packages/commonwealth/client/scripts/views/components/Header/LoginSelectorMenuLeft.tsx

```diff
@@ -10,6 +10,7 @@ import { CWText } from 'views/components/component_kit/cw_text';
 import { isWindowMediumSmallInclusive } from 'views/components/component_kit/helpers';
 import { UserBlock } from 'views/components/user/user_block';
 import { LoginModal } from 'views/modals/login_modal';
+import { CWModal } from '../component_kit/new_designs/CWModal';
```

Comment:
'Modal' is defined but never used

Reply:
[Addressed](https://github.com/hicommonwealth/commonwealth/commit/3794fc26f0acfd7ed8bb37b4a0cdd62bdf6d0dc3).